_i18n/en.yml

```diff
@@ -801,7 +801,24 @@ roadmap:
   p2pool: P2Pool released
   haveno: Haveno
 
-
+  ringsize-sera-tryp: increase ring size to 100+ through Seraphis or Triptych
+  seraphis-jamtis: Seraphis & jamtis
+  ospead: OSPEAD
+  released-0-17-3-0-cli: CLI 0.17.3.0 released
+  released-0-17-3-0-gui: GUI 0.17.3.0 released
+  released-0-17-3-1-gui: GUI 0.17.3.1 released
+  released-0-17-3-2-gui: GUI 0.17.3.2 released
+  released-0-17-3-2-cli: CLI 0.17.3.2 released
```

Comment:
I would list only major versions in the roadmap or it will get cluttered. The roadmap should contain only major achievements.